tasks/install.yml

```diff
@@ -0,0 +1,80 @@
+---
+
+- name: Download latest wordpress version
+  get_url:
+    url: https://wordpress.org/latest.zip
+    dest: /tmp/wordpress_latest.zip
+
+- name: Unarchive wordpress zip to /var/www
+  unarchive:
+    src: /tmp/wordpress_latest.zip
+    dest: /var/www/
+    owner: www-data
+    group: www-data
+
+- name: Copy theme
+  unarchive:
+    src: "{{ inventory_hostname }}/{{ wordpress_template }}.zip"
+    dest: /var/www/wordpress/wp-content/themes/
+    directory_mode: yes
+    mode: 0755
+    owner: www-data
+    group: www-data
+  when: wordpress_import_template
+
+- name: remove anonymous user account
```

Comment:
Capitalize.
Also, is the username actually an empty string?

Reply:
yes it is